### PR TITLE
chore: Update tekton pruner logic

### PR DESCRIPTION
**Pull request by Rolika4**
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: charts/pipelines-library/templates/resources/pruner/configmap.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {{ if .Values.tekton.pruner.create }}
 ---
-kind: Secret
+kind: ConfigMap
 apiVersion: v1
 type: Opaque
```
Comment (NikolayMarusenko) [inline charts/pipelines-library/templates/resources/pruner/configmap.yaml:5]: please check this field

File: charts/pipelines-library/templates/resources/pruner/rbac.yaml [hunk 2]
```diff
@@ -36,7 +37,6 @@ rules:
       - watch
       - delete
     resources:
-      - pods
       - persistentvolumeclaims
```
Comment (NikolayMarusenko) [inline charts/pipelines-library/templates/resources/pruner/rbac.yaml:40]: do we need pvc?

File: charts/pipelines-library/scripts/tekton-prune.sh [hunk 3]
```diff
@@ -79,51 +62,29 @@ prune_resources() {
...
   echo 'Get active pipelineruns'
   active_pipelineruns=$(get_active_pipelineruns)
-  echo "active pipelineruns: $active_pipelineruns"
+  echo "Running pipelineruns: $active_pipelineruns"
 
-  echo "Get pipelineruns completed recently (in the last ${RECENT_MINUTES_PODS} minutes)"
-  recent_pipelineruns_pods=$(get_recent_pipelineruns "${RECENT_MINUTES_PODS}")
-  echo "recent pipelineruns: $recent_pipelineruns_pods"
+  echo "Get pipelineruns completed recently (in the last ${RECENT_MINUTES_LOGS} minutes)"
+  echo "Pipelineruns completed recently: $recent_pipelineruns"
```
Comment (Artem-Zahumonnyi) [inline charts/pipelines-library/scripts/tekton-prune.sh:74]: recent_pipelineruns - what does this variable contain?

File: charts/pipelines-library/templates/resources/pruner/configmap.yaml (hunk 1 above)
Comment (Rolika4) [reply charts/pipelines-library/templates/resources/pruner/configmap.yaml:5]: change type for script

File: charts/pipelines-library/templates/resources/pruner/rbac.yaml (hunk 2 above)
Comment (Rolika4) [reply charts/pipelines-library/templates/resources/pruner/rbac.yaml]: removed

File: charts/pipelines-library/scripts/tekton-prune.sh (hunk 3 above)
Comment (Rolika4) [reply charts/pipelines-library/scripts/tekton-prune.sh]: removed

**Review by NikolayMarusenko — APPROVED** [review]

**Pull request merged**